install.sh

```diff
@@ -1,45 +1,62 @@
-#!/bin/bash
+#!/bin/bash -x
+
+# Source setting.sh
+source /vagrant/settings.sh
+
+# Fetching LAN_IP and network address
+if [[ ${STANDALONE,,} != yes ]]; then
+   LAN_IP=$(ip addr | grep -Po -m3 'inet \K[\d.]+' | tail -n1)
+fi
+NETWORK="$(echo $LAN_IP | cut -d'.' -f1-3)"
 
 # Updating packages
 apt --yes update
 apt --yes upgrade
 
-# Installing dependencies
+# Installing packages
 apt --yes --install-recommends install dnsmasq 
 apt --yes --install-recommends install ltsp-server
+# Setting type of user interface with a boot parameter - https://www.debian.org/releases/jessie/i386/ch05s03.html
 DEBIAN_FRONTEND=noninteractive apt --yes --install-recommends install ltsp-client
 apt --yes install epoptes epoptes-client
 
 # Installing debian edu packages
 source /vagrant/edu-packages/edu.sh
 
+# Installing xfce desktop environment
+apt --yes install task-xfce-desktop 
```

Comment:
It would be better if you used something similar to LinuxMint for the box, instead of installing the xfce desktop separately. For example this one: https://linuxmint.com/download_lmde.php

Reply:
Linux Mint Debian Edition looks very promising. But there are no vagrant boxes for it. Building one can take a lot of time. Even if we build one testing would again take a lot time.

I understand image with preinstalled gui would be the best thing. but `task-xfce-desktop` is doing a great job out of all the DE available. I tested many of them

Reply:
LMDE 3 Cindy has been released: https://blog.linuxmint.com/?p=3629